src/actor/Claimer.spec.ts

```diff
@@ -355,6 +387,28 @@ describe('Claimer', () => {
           })
         ).rejects.toThrowError(ERROR_PE_CREDENTIAL_MISSING())
       })
+      it('Should throw, if PE is allowed, but claimer has no PE-enabled identity', async () => {
+        const claimerWithoutPE = await Identity.buildFromURI('//bob', {
```

Comment:
Not important, but this creates a different identity than from the `//Bob` URI.

```suggestion
        const claimerWithoutPE = await Identity.buildFromURI('//Bob', {
```

Reply:
fixed in ea38166a4ce1b9c442db0264098e7f7ff080847f